model/adapter.py

```diff
@@ -723,6 +725,32 @@ def query_fun(rxn):
             reaction.bounds = 0, upper_bound
         return reaction
 
+    def minimize_distance(self):
+        """Replaces fluxomics measurements with the minimized distance"""
+        reaction_measurements = [m for m in self.measurements if m['type'] == 'reaction']
+        observations = {m['id']: np.mean(m['measurements']) for m in reaction_measurements}
+
+        uncertainties = {
+            m['id']: np.std(m['measurements']) if len(m['measurements']) >= 3 else 1
```

Comment:
See my point above, there is an equivalent `np.nanstd`. For either method you should probably supply the argument `ddof=1`.